charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -18,7 +18,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 12.5.0
+version: 12.6.0
```

Comment:
lgtm, but would it be an increment to minor or patch?

Reply:
I wasn't sure 100% myself. As mentioned in the PR description, I though _minor_ was appropriate here, what do you think?

> I'm bumping the _minor_ part of the chart's semver because the change does have an effect on the current's default.